ang/civicase-base/directives/checkbox.directive.html

```diff
@@ -0,0 +1,20 @@
+<div class="civicase__case-filter-form-elements">
```

Comment:
`civicase__case-filter-form-elements` this class should not be part of generic checkbox component.

Reply:
Also `civicase__checkbox` is already used in many places, is it possible to use this component in all those places?
Not urgent, just asking the possibility,

Reply:
Thanks @deb1990 , removed the reference to `civicase__case-filter-form-elements` from the checkbox component and updated the search directive so it uses the component all over.

I was looking at other places where it should be replaced but didn't do it because it requires more consideration. For example the case overview contains code for the checkbox component:
https://github.com/compucorp/uk.co.compucorp.civicase/commit/55e26f07c17664b1ecf141c2f4cc5118852ab00c#diff-0d72a281a3c4a2b7cb75fe22ddd0f89bd90cea0fc484930fd481f66587ede5daR256-R265

Ideally we need to remove all those references, but it would require running backstop (which is easier) and doing manual tests. 

We should remove these as we find them.